src/Categories/Examples/Monad/Duration.agda

```diff
@@ -0,0 +1,99 @@
+{-# OPTIONS --safe --without-K #-}
+module Duration where
+
+open import Level
+
+open import Function using (_$_)
+open import Function.Equality using (Π; _⟨$⟩_; cong)
+
+open import Relation.Binary
+open import Relation.Binary.Indexed.Heterogeneous.Construct.Trivial
+
+open import Data.Product using (Σ; _,_; _×_; proj₁; proj₂)
+open import Algebra using (Monoid; IsMonoid; IsSemigroup; IsMagma)
+
+open import Categories.Category
+open import Categories.Category.Instance.Setoids
+open import Categories.Functor renaming (id to idF)
+open import Categories.NaturalTransformation using (NaturalTransformation)
+open import Categories.Monad using (Monad)
+
+open import Data.Nat.Properties renaming (+-0-monoid to +-0-nat-monoid)
+open import Data.Integer.Properties renaming (+-0-monoid to +-0-integer-monoid)
+open import Data.Rational.Properties renaming (+-0-monoid to +-0-rational-monoid)
+
+open Monoid public
+
+monoid×-₀ : Monoid 0ℓ 0ℓ → Setoid 0ℓ 0ℓ → Setoid 0ℓ 0ℓ
+monoid×-₀ record { Carrier = Carrierₘ ; _≈_ = _≈ₘ_ ; isMonoid = isMonoid } record { Carrier = Carrier ; _≈_ = _≈_ ; isEquivalence = isEquivalence } = record
```

Comment:
We follow the same style as in the library, and don't pattern-match on record like this (too noisy). The better solution is to make local modules using a `where` clause (with renaming if you want).